spec/player_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require './lib/player.rb'
+
+describe Player do
+  let(:new_player) { Player.new }
+  describe '#additional_method' do
+    it '1 expected to return 1' do
```

Comment:
- [ ] **[REQUIRED]** Test failing due to an invalid number of arguments passed. Please fix it.